src/components/EReceiptWithSizeCalculation.tsx

```diff
@@ -42,14 +50,19 @@ function EReceiptWithSizeCalculation(props: EReceiptWithSizeCalculationProps) {
                     styles.w100,
                     styles.h100,
                     {transform: `scale(${scaleFactor}) ${styles.translateZ0.transform as string}`, transformOrigin: 'top left'},
-                    props.shouldUseAspectRatio && {aspectRatio: eReceiptAspectRatio},
+                    (props.receiptType === 'perDiem' ? true : props.shouldUseAspectRatio) && {aspectRatio: eReceiptAspectRatio},
```

Comment:
We can simplify this code and use the condition like this:

```js
(props.receiptType === 'perDiem' || props.shouldUseAspectRatio) && {aspectRatio: eReceiptAspectRatio}
```

Reply:
Yes but then ESLint flags it as a usage of `||` instead of `??`

So, what should be done? Should we silence the ESLint issue for the cleaner version, or should we keep using the longer?

Reply:
Yeah, I think it’s fine to keep it as is , no need to disable the lint warning.